Userland/Utilities/less.cpp

```diff
@@ -181,8 +179,8 @@ class Pager {
     {
         // First, we get the current size of the window.
         struct winsize window;
-        if (ioctl(STDOUT_FILENO, TIOCGWINSZ, &window) == -1) {
-            perror("ioctl(2)");
+        if (auto maybe_error = Core::System::ioctl(STDOUT_FILENO, TIOCGWINSZ, &window); maybe_error.is_error()) {
+            warnln("ioctl(2): {}", strerror(maybe_error.error().code()));
```

Comment:
not in the scope of this PR, but error should really have a way to get the error string, or strerror() if it's not present.

Reply:
Agreed. The closest we can get to this at the moment is something like: `warnln("{}", maybe_error.release_error())`, but that introduces some formatting that we don't want here.  The above also changes behavior slightly, as it no longer prints to the debug console.